### PR TITLE
Or 1474 add core utils in tokamak directory

### PR DESCRIPTION
**Pull request by rlgns98kr**
- add core-utils in tokamak directory
- remove devnet-legacy

About this, I think we need tokamak/core-utils to apply custom name (LegacyERC20ETH -> LegacyERC20NativeToken...)
If do you have any other opinions, please let me know!

If this PR merge, I will merge `main` in the `OR-1257-Update-smart-contracts-for-deposit-TON-in-L1`

Thank you! 🙇🏼‍♂️

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -147,20 +138,9 @@ devnet-clean:
 	docker volume ls --filter name=ops-bedrock --format='{{.Name}}' | xargs -r docker volume rm
 .PHONY: devnet-clean
 
-devnet-legacy-clean:
```
Comment (nguyenzung) [inline Makefile:150]: I guess we undo some options here, is it right? :D

Comment (rlgns98kr) [reply Makefile:150]: Yes, I'm sorry for my lack of explanation.
I think we can't supply devnet-legacy (ETH native) if we change the name of LEGACY_ERC20_ETH in our `op-chain-ops`. So I asked that Is it okay if we remove devnet-legacy to contracts team.

**Comment by rlgns98kr** [comment]
@tokamak-network/layer2-core @boohyung @ohbyeongmin @AaronLee22 
I'm sorry for making confusions. Can you check my code?

File: packages/tokamak/core-utils/src/optimism/constants.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  DeployerWhitelist: '0x4200000000000000000000000000000000000002',
+  L2CrossDomainMessenger: '0x4200000000000000000000000000000000000007',
+  GasPriceOracle: '0x420000000000000000000000000000000000000F',
+  L2StandardBridge: '0x4200000000000000000000000000000000000010',
+  SequencerFeeVault: '0x4200000000000000000000000000000000000011',
+  OptimismMintableERC20Factory: '0x4200000000000000000000000000000000000012',
+  L1BlockNumber: '0x4200000000000000000000000000000000000013',
+  L1Block: '0x4200000000000000000000000000000000000015',
+  LegacyERC20ETH: '0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000',
+  WETH9: '0x4200000000000000000000000000000000000006',
```
Comment (nguyenzung) [inline packages/tokamak/core-utils/src/optimism/constants.ts:22]: I think we should update WETH9 -> WNativeToken, and add ETH to predeploy

Comment (nguyenzung) [reply packages/tokamak/core-utils/src/optimism/constants.ts:22]: Oh, the target branch is main, so it is OK

Comment (rlgns98kr) [reply packages/tokamak/core-utils/src/optimism/constants.ts:22]: It's correct! Thank you, Brave! I will do it on my next ticket!

Comment (rlgns98kr) [reply packages/tokamak/core-utils/src/optimism/constants.ts:22]: I will merge main in OR-1257(your branch). So I think I can work for it at that time as well!

**Review by nguyenzung — APPROVED** [review]
Well done, Steven! ^^

**Pull request merged**